python/ray/tune/integration/pytorch_lightning.py

```diff
@@ -5,13 +5,193 @@
 from ray import tune
 
 import os
+from importlib_metadata import version
+from packaging.version import parse as v_parse
 
 logger = logging.getLogger(__name__)
 
+ray_pl_use_master = v_parse(version("pytorch_lightning")) >= v_parse("1.6")
```

Comment:
nit: Would rename this to something more descriptive since master may change in the future (e.g. `use_ptl_1_6_api`)